projects/igniteui-angular/src/lib/grids/common/grid.interface.ts

```diff
@@ -461,13 +461,15 @@ export interface GridType extends IGridDataBindable {
     rowEditingOverlay: IgxToggleDirective;
     totalRowsCountAfterFilter: number;
     _totalRecords: number;
-
     pagingMode: GridPagingMode;
+    /** @hidden */
```

Comment:
also internal, but regardless this should clear it out from the API docs and the rest of this interface due for an overhaul anyway